LanguageFeatures/Views/syntax_A01_t01.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2022, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion A rule for <viewDeclaration> is added to the grammar, along with
+/// some rules for elements used in view declarations:
+///
+/// <viewDeclaration> ::=
+///   'view' 'class' <typeIdentifier> <typeParameters>?
+///       <viewPrimaryConstructor>?
+///       <interfaces>?
+///   '{'
+///     (<metadata> <viewMemberDeclaration>)*
+///   '}'
+///
+/// <viewPrimaryConstructor> ::=
+///   '(' <type> <identifier> ')'
+///
+/// <viewMemberDeclaration> ::=
+///   <classMemberDefinition>
+/// The token `view` is made a built-in identifier.
+///
+/// @description Checks that it is a compile-time error to declare a function
+/// named `view`
```

Comment:
No, that would actually not be an error. I've specified that `view` is a built-in identifier, which means that it would be an error to declare a type with that name, but other things (a function, a local variable, etc, anything-not-an-type) would be ok.